addons/website/views/website_templates.xml

```diff
@@ -1922,6 +1891,19 @@
     </xpath>
 </template>
 
+<template id="website.placeholder_header_call_to_action" name="Placeholder Header Call to Action"/>
+<template id="website.header_call_to_action" inherit_id="website.placeholder_header_call_to_action" name="Placeholder Header Call to Action" active="True">
+    <xpath expr="." position="inside">
+        <div t-attf-class="oe_structure oe_structure_solo #{_div_classes}">
+            <section class="s_text_block" data-snippet="s_text_block" data-name="Text">
+                <div class="container">
+                    <a href="/contactus" class="btn btn-primary ml-lg-4 ml-2">Contact Us</a>
+                </div>
+            </section>
+        </div>
+    </xpath>
+</template>
```

Comment:
Problem in editor, strange list label:
![Screenshot from 2021-06-29 14-12-57](https://user-images.githubusercontent.com/10338094/123795329-32b81500-d8e4-11eb-9169-e237fd9b99fa.png)

Reply:
I don't have this problem, I think it's a resource that hasn't uploaded well 
![Uploading Capture d’écran 2021-06-29 à 16.06.06.png…]()